rds-deployment-ha-gateway/azuredeploy.json

```diff
@@ -347,7 +360,7 @@
                         "typeHandlerVersion": "2.76",
                         "autoUpgradeMinorVersion": true,
                         "settings": {
-                            "ModulesUrl": "[concat(variables('assetLocation'),'/Configuration.zip')]",
+                            "ModulesUrl": "[concat(parameters('_artifactsLocation'),'DSC/Configuration.zip',parameters('_artifactsLocationSasToken'))]",
```

Comment:
See: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#creating-uris for how to create the uris

Reply:
fixed